src/partials/_about.hbs

```diff
@@ -2,20 +2,67 @@
   <h2 class="c-about__header">Perfect for you and your&nbsp;child</h2>
   <p class="c-about__text">Endearing animals and engaging games for them. Classic design and peace of mind for&nbsp;you.</p>
   {{> _animal--lion}}
-  <svg width="160" height="160" class="c-lion__back parallax-one">
+  <svg width="160" height="160" class="c-lion__back" id="c-lion__back">
     <circle cx="80" cy="80" r="80" />
   </svg>
-  <svg class="c-lion__element" id="c-lion__element__1">
-    <polygon points="0,15 9,0 18,15" />
+  <svg width="18" height="15">
+    <symbol id="triangle">
+      <polygon points="0,15 9,0 18,15" />
+    </symbol>
   </svg>
-  <svg class="c-lion__element" id="c-lion__element__2">
-    <polygon points="0,15 9,0 18,15" />
+  <svg class="c-lion__element" id="c-lion__element--1">
+    <use xlink:href="#triangle"></use>
   </svg>
-  <svg class="c-lion__element" id="c-lion__element__3">
-    <polygon points="0,15 9,0 18,15" />
+  <svg class="c-lion__element" id="c-lion__element--2">
+    <use xlink:href="#triangle"></use>
   </svg>
-  <svg class="c-lion__element" id="c-lion__element__4">
-    <polygon points="0,15 9,0 18,15" />
+  <svg class="c-lion__element" id="c-lion__element--3">
+    <use xlink:href="#triangle"></use>
+  </svg>
+  <svg class="c-lion__element" id="c-lion__element--4">
+    <use xlink:href="#triangle"></use>
+  </svg>
+  {{> _animal--owl}}
+  <svg width="160" height="160" class="c-owl__back" id="c-owl__back">
+    <rect width="250" height="200">
+  </svg>
+  <svg width="18" height="15">
+    <symbol id="half-circle">
+      <path class="cls-1" d="M9.08 18.17A9.085 9.085 0 0 1 9.08 0v18.17z"/>
+    </symbol>
+  </svg>
+  <svg class="c-owl__element" id="c-owl__element--1">
+    <use xlink:href="#half-circle"></use>
+  </svg>
+  <svg class="c-owl__element" id="c-owl__element--2">
+    <use xlink:href="#half-circle"></use>
+  </svg>
+  <svg class="c-owl__element" id="c-owl__element--3">
+    <use xlink:href="#half-circle"></use>
+  </svg>
+  <svg class="c-owl__element" id="c-owl__element--4">
+    <use xlink:href="#half-circle"></use>
+  </svg>
+  {{> _animal--elephant}}
```

Comment:
Do you have fallback in place for browser that do not support inline-SVGs, or is the intent to have no images display?